tilings/algorithms/minimal_gridded_perms.py

```diff
@@ -301,13 +285,13 @@ def get_upward_closure(self, gps: GPTuple) -> GPTuple:
             self.upward_closures[gps] = res
         return res
 
-    def get_cells_to_try(
-                         self,
-                         gp: GriddedPerm,
-                         gps: GPTuple,
-                         last_cell: Cell,
-                         try_localised: bool
-                        ) -> Iterator[Cell]:
+    def _get_cells_to_try(
+                          self,
+                          gp: GriddedPerm,
+                          gps: GPTuple,
+                          last_cell: Cell,
+                          try_localised: bool
+                         ) -> Iterator[Cell]:
```

Comment:
The output type is incorrect as it returns a `Tuple[Cell, bool]`

Reply:
Fixed.